### DISCUSSION TITLE
umi4 如何取消以 package.json 中的 name 为默认路由前缀

### DISCUSSION THREAD

**Discussion by tpc-ht** [umijs/umi#11411] [Q&A] [answered]
![image](https://github.com/umijs/umi/assets/59469126/637cdb19-2f98-4c09-a76f-5cf304f07415)
![image](https://github.com/umijs/umi/assets/59469126/89c4dcfe-8684-4da1-a93d-fc4cfcae717c)
只有 ` /wechat/home ` 可以访问，` /home  ` 404
![image](https://github.com/umijs/umi/assets/59469126/d94d5adf-5132-45bb-8ac7-b48afcb570fa)

**Comment by fz6m** [comment, marked as answer]
子应用是这样的，独立开发的时候自带 `package.json` 的 name ，可以在本地开发的时候不开 qiankun 选项，构建开就行。

**Reply by tpc-ht** [reply]
谢谢大佬，非常有用
